array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,10 @@
+/* exported compact */
+function compact(array) {
+  var newArr = [];
+  for (var i = 0; i < array.length; i++) {
+    if (!array[i] === false) {
```

Comment:
You don't need to do `=== false`. A conditional will always either evaluate to true or false, so you can just check if the current iteration is truthy or falsy
```
function compact(array) {
  var output = [];
  for (var i = 0; i < array.length; i++) {
    var element = array[i];
    if (element) output.push(element);
  }
  return output;
}
```